views/course_view.xml

```diff
@@ -0,0 +1,45 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record model="ir.actions.act_window" id="open_academy_course_action">
+        <field name="name">Courses</field>
+        <field name="res_model">course</field>
+        <field name="view_mode">tree,kanban,form</field>
+    </record>
+
+    <record model="ir.ui.view" id="open_academy_course_view_action">
+        <field name="name">List courses</field>
+        <field name="model">course</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="title" string="Courses" />
+                <field name="description" string="Description" />
+            </tree>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="open_academy_course_form_action">
+        <field name="name">form courses</field>
+        <field name="model">course</field>
+        <field name="arch" type="xml">
+            <form>
+                <sheet string="Form course">
+                    <notebook>
+                        <page name="Course information" string="Course information">
+                            <group col="4">
+                                <field name="title" />
+                                <newline />
+                            </group>
```

Comment:
The exercise only asks to add the description field to the page. Please remove the title field from here and add it outside the page.